bitnami/redis/templates/configmap-scripts.yaml

```diff
@@ -12,6 +12,8 @@ data:
 {{- if and .Values.cluster.enabled .Values.sentinel.enabled }}
   start-node.sh: |
     #!/bin/bash
+    set -x
```

Comment:
This is for debugging purposes, right? Please remove it

Reply:
Yes. Removed.